packages/TelegramClient-Core.package/TCCChatsHandler.class/instance/addChat..st

```diff
@@ -3,11 +3,7 @@ addChat: aChatEvent
 
 	| chat |
 	
-	chat := TCCChat new
-		id: ((aChatEvent at: 'chat') at: 'id');
-		title: ((aChatEvent at: 'chat') at: 'title');
-		canSendMessages: (((aChatEvent at: 'chat') at: 'permissions') at: 'can_send_messages');
-		lastMessage: ''.
+	chat := TCCChat newFromNewChatEvent: aChatEvent.
 
 	self app chats
 		add: chat;
```

Comment:
Here the local variable `chat` seems somewhat unnecessary, maybe just directly create the chat in the setter

Reply:
yes